### PR TITLE
feat(liniting): add overlay to subprocess view

### PR DESCRIPTION
**Pull request by marstamm**
related to https://github.com/bpmn-io/internal-docs/issues/340
recording (56)

### CODE REVIEW

**Comment by marstamm** [comment]
This is ready for review now :)

**Comment by nikku** [comment]
I'll have a look into this one.

File: lib/Linting.js [hunk 1]
```diff
@@ -300,14 +300,21 @@ Linting.prototype._createElementIssues = function(elementId, elementIssues) {
...
   var position;
 
-  if (element === this._canvas.getRootElement()) {
+  var isRoot = !element.parent;
+  if (isRoot) {
     menuPosition = 'bottom-right';
 
     position = {
-      top: 20,
+      top: 50,
```
Comment (nikku) [inline lib/Linting.js:317]: Is this to account for a potential breadcrumb?

image

What about making it adaptable (i.e. react to weather the [`.bjs-breadcrumbs-shown`] is there).

Comment (marstamm) [reply lib/Linting.js:317]: Good point, I'll add that! The original PR was created before we had proper css classes for that and I did not think about it when rebasing it

Comment (marstamm) [reply lib/Linting.js:324]: Reacting to the breadcrumb css felt a bit clunky when all overlays are positioned absolutely. I added the cases:
  - root is process
  - root is SubProcess

as separate options with 81b140bd217d77ddf8331c7ec6f197df9cf2fa55

**Review by nikku — APPROVED** [review]
Very nice.

**Pull request merged**